module/android/build_steps.json

```diff
@@ -84,54 +84,23 @@
                 "element": {
                     "tag": "service",
                     "attributes": {
-                        "android:name": "io.trigger.forge.android.modules.parse.StaffbaseMessagingService"
```

Comment:
Diff is a bit messed up here. 
The real change is just a deletion of [these lines](https://github.com/Staffbase/trigger.io-parse/blob/master/module/android/build_steps.json#L103-L134) and the addition of `"android:exported": "false"` [here](https://github.com/Staffbase/trigger.io-parse/blob/master/module/android/build_steps.json#L87).